app/services/foreman/unattended_installation/host_verifier.rb

```diff
@@ -0,0 +1,76 @@
+module Foreman
+  module UnattendedInstallation
+    class HostVerifier
```

Comment:
To me a class feels very heavy weight for what feels to me could be a function. That might be my personal preference.